lib/internal/abort_controller.js

```diff
@@ -328,7 +349,10 @@ class AbortController {
   /**
    * @param {any} [reason]
    */
-  abort(reason = new DOMException('This operation was aborted', 'AbortError')) {
+  abort(reason) {
+    if (reason === undefined) {
+      reason = lazyDOMException();
```

Comment:
Since we are not creating a new error everytime abort is called, 2 different usages in the code throwing this error, might have a different stack trace. Is this compliant with the spec?

Reply:
ah good point, not sure about that one thing we could do maybe is move this lazy loading inside the functions themselves. that should ensure correct trace?